scripts/duplicate_detector.py

```
@@ -0,0 +1,275 @@
+    )
+
+    if is_manual_trigger:
+        logging.info('Manual trigger detected. Batch mode.')
+        start_issue = int(start_issue_env) if start_issue_env else 1
+        end_issue = int(end_issue_env) if end_issue_env else 999999999
+        logging.info('Fetching all open issues for batch processing...')
+        all_issues = get_all_open_issues(repo, headers)
+
+        for iss in all_issues:
+            if start_issue <= iss['number'] <= end_issue:
+                issues_to_triage.append(iss)
```

Ruff (0.15.20)

[warning] Use `list.extend` to create a transformed list

(PERF401)